Misc/NEWS.d/next/Core and Builtins/2017-09-06-15-25-59.bpo-31373.dC4jd4.rst

```diff
@@ -0,0 +1,6 @@
+Fix several possible instances of undefined behavior due to floating-point
+demotions.
+
+# Write your Misc/NEWS entry below.  It should be a simple ReST paragraph. #
```

Comment:
You are not supposed to commit this part :-)

Reply:
hmm, blurb hiccup I guess